tests/e2e/framework/istioctl.go

```diff
@@ -131,6 +131,11 @@ func (i *Istioctl) CreateRule(rule string) error {
 		i.namespace, i.istioNamespace, rule))
 }
 
+// CreateMixerRule create new rule(s)
+func (i *Istioctl) CreateMixerRule(scope, subject, rule string) error {
+	return i.run(fmt.Sprintf("-n %s mixer rule create %s %s -f %s", i.namespace, scope, subject, rule))
```

Comment:
`--istioNamespace %s` is needed to specify namespace of istio system components (see CreateRule above for example).